### PR TITLE
Update Runtime script

### LINKED ISSUE #2: copy notebooks directory into a named folder
Now the notebooks are copied into `notebooks-<timestamp>`
If you start different images, this is not fun.
Let's start with inferring the folder name from the git repo that is used to build the image. 
If that's not easy, use an environment variable

### PR DESCRIPTION
**Pull request by 4n4nd**
Change new directory name to repo name
Auto generate README.md

Closes #2 #3

### CODE REVIEW

File: .s2i/bin/run [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+touch $readme_location
+echo "# $GIT_REPO_NAME" > $readme_location
+echo "#### Version: $GIT_TAG" >> $readme_location
+echo "#### Author: $GIT_TAG_AUTHOR" >> $readme_location
+echo "" >> $readme_location
+echo "$GIT_REPO_DESC <br>" >> $readme_location
+echo "" >> $readme_location
+echo "### Source: <br>" >> $readme_location
+echo "This notebook image has been created from notebooks available at: " >> $readme_location
+echo "[$GIT_TAG_COMMIT_SHA](${GIT_REPO_URL:0:-4}/tree/$GIT_TAG_COMMIT_SHA/notebooks)" >> $readme_location
```
Comment (harshad16) [inline .s2i/bin/run]:
```suggestion
echo "generating README.md"
cat <<EOF > $notebooks_location/README.md
# "$GIT_REPO_NAME"

#### Version: "$GIT_TAG"
#### Author: "$GIT_TAG_AUTHOR"

"$GIT_REPO_DESC"<br>

### Source: <br>
This notebook image has been created from notebooks available at:
[$GIT_TAG_COMMIT_SHA](${GIT_REPO_URL:0:-4}/tree/$GIT_TAG_COMMIT_SHA/notebooks)
EOF
```
Newline can be easily incorporated in above line, with additional command execution.
I dont have major experience with shell script. WDYT?

**Review by harshad16 — APPROVED** [review]
Apart from the small comment rest LGTM.

File: .s2i/bin/run (hunk 1 above)
Comment (4n4nd) [reply .s2i/bin/run]: Yeah that looks a lot better :+1:

**Comment by harshad16** [comment]
@4n4nd  This PR is good to merge. Already approved it ;)
I don't have write access to merge it myself.

**Review by harshad16 — APPROVED** [review]

**Comment by 4n4nd** [comment]
> @4n4nd This PR is good to merge. Already approved it ;)
> I don't have write access to merge it myself.

oh oops

**Pull request merged**